configure.php

```diff
@@ -129,6 +129,38 @@ function remove_project_files() {
 	echo 'Removed .buddy, buddy.yml, CHANGELOG.md, .deployignore, .editorconfig, .gitignore, .gitattributes, .github and LICENSE files.' . PHP_EOL;
 }
 
+function rollup_phpcs_to_parent( string $parent_file, string $local_file, string $plugin_name, string $plugin_domain ) {
+	$config = '<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="' . $plugin_name . ' Configuration" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
+  <description>PHP_CodeSniffer standard for ' . $plugin_name . '</description>
+
+  <!-- DO NOT ADD ADDITIONAL RULES TO THIS FILE. Modifications belong in the root-level configuration. -->
+
+  <!-- Include Root Rules -->
+  <rule ref="' . $parent_file . '" />
+
+  <rule ref="WordPress.WP.I18n">
+    <properties>
+      <!--
+      Verify that the text_domain is set to the desired text-domain.
+      Multiple valid text domains can be provided as a comma-delimited list.
+       -->
+      <property name="text_domain" type="array" value="' . $plugin_domain . '" />
+    </properties>
+  </rule>
+
+  <rule ref="WordPress.NamingConventions.PrefixAllGlobals">
+    <properties>
+      <property name="prefixes" type="array" value="' . $plugin_domain . ',ai_,_ai" />
```

Comment:
The `ai` prefixes are unique to Alley – and I don't see any globals prefixed with them in the plugin. I think we can safely delete these.

Also, the global prefix is snakecase.

```suggestion
      <property name="prefixes" type="array" value="' . str_replace( '-', '_', $plugin_domain ) . '" />
```

Reply:
The domain in the plugin is already `create-wordpress-plugin` -- the domain should be kebab case (but it really is the same difference 😆 ) https://developer.wordpress.org/themes/functionality/internationalization/#text-domain